docs/DopplLibrary.md

```diff
@@ -0,0 +1,439 @@
+
+First, add in the `dopplArchVer` and `dopplRetroVer` constants that we used in
+`SOAndroid`:
+
+```groovy
+def dopplArchVer = "1.0.0.0-rc1"
+def dopplRetroVer = "2.3.0.7"
+```
+
+Then, replace the `dependencies` closure in `service-api/build.gradle` with the
+following:
+
+```groovy
+dependencies {
+  api                 "android.arch.lifecycle:runtime:1.0.3"
+  doppl               "co.doppl.android.arch.lifecycle:runtime:$dopplArchVer"
+  api                 "android.arch.lifecycle:extensions:$archVer"
+  doppl               "co.doppl.android.arch.lifecycle:extensions:$dopplArchVer"
+  implementation      "io.reactivex.rxjava2:rxjava:2.1.5"
+  doppl               "co.doppl.io.reactivex.rxjava2:rxjava:2.1.5.0"
+  implementation      "io.reactivex.rxjava2:rxandroid:2.0.1"
+  doppl               "co.doppl.io.reactivex.rxjava2:rxandroid:2.0.1.2"
+  implementation      "com.squareup.retrofit2:retrofit:$retroVer"
+  implementation      "com.squareup.retrofit2:converter-gson:$retroVer"
+  implementation      "com.squareup.retrofit2:adapter-rxjava2:$retroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:retrofit:$dopplRetroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:converter-gson:$dopplRetroVer"
+  doppl               "co.doppl.com.squareup.retrofit2.urlsession:adapter-rxjava2:$dopplRetroVer"
+
+  testImplementation  "junit:junit:4.12"
+  testImplementation  "co.doppl.lib:androidbasetest:0.8.5"
+  testDoppl           "co.doppl.lib:androidbasetest:0.8.5.0"
+}
+```
+
+As before, we set up `doppl` equivalents for everything that we had been using
+for Android, plus add some Doppl dependencies specific for our unit tests.
+
+## Step #8: Perform the Doppl Conversion
+
+Next, open the Gradle tool in Android Studio, and in the `:service-api` set of
+`doppl` tasks, run `dopplBuild`:
+
+![`dopplBuild` Task in Android Studio](./DopplLibrary-1.png)
+
+As before, this will convert your Java into Objective-C and package them into
+pods for easy use from iOS projects in Xcode. And, for this sample project,
+everything should build cleanly:
+
+![Results of `dopplBuild` Task in Android Studio](./DopplLibrary-2.png)
+
+The following steps will need to be performed on a macOS machine set up
+for iOS development. If you were doing the preceding steps on some other
+machine, transfer your project over to the macOS machine now.
+
+## Step #9: Create the Library Test Project
+
+You now need to create an Xcode project for the tests. As before, these can
+reside anywhere, but it is a bit simpler if they
+are part of the overall project. So, in this tutorial, you will create an
+`iosServiceApiTest/` project off of the main project root directory.
+
+To do this:
+
+- Start Xcode.
+
+- If you come up to the welcome dialog, choose "Create a new Xcode project"
+(see below screenshot). If you were already in one Xcode project, choose
+File > New > Project... from the main menu.
+
+![Xcode Welcome Dialog](./AddingDoppl-4.png)
```

Comment:
this is showing up as a link instead of an image when I view the readme preview

Reply:
That image is shared with the adding-Doppl-to-a-project document, so it is not showing up in this topic branch, as that's in a different topic branch.